modules/monitoring-silencing-alerts.adoc

```diff
@@ -27,36 +27,64 @@ To silence a specific alert:
 
 * In the *Administrator* perspective:
 
-. Navigate to the *Observe* -> *Alerting* -> *Alerts* page of the {product-title} web console.
+. Go to *Observe* -> *Alerting* -> *Alerts* in the {product-title} web console.
 
-. For the alert that you want to silence, select the {kebab} in the right-hand column and select *Silence Alert*. The *Silence Alert* form will appear with a pre-populated specification for the chosen alert.
+. For the alert that you want to silence, click {kebab} and select *Silence Alert* to open the *Silence Alert* page with a default configuration for the chosen alert.
 
-. Optional: Modify the silence.
+. Optional: Change the default configuration details for the silence.
++
+[NOTE]
+====
+You must add a comment before saving a silence.
+====
 
-. You must add a comment before creating the silence.
-
-. To create the silence, select *Silence*.
+. To save the silence, click *Silence*.
 
 * In the *Developer* perspective:
 
-. Navigate to the *Observe* -> *<project_name>* -> *Alerts* page in the {product-title} web console.
+. Go to *Observe* -> *<project_name>* -> *Alerts* in the {product-title} web console.
+
+. If necessary, expand the details for the alert by selecting *>* next to the alert name.
```

Comment:
as I checked on developer console, the details are expanded, not collapsed. keep this description is also fine